GooglePlayInstant/Samples/TestApp/Scripts/TestApp.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+
+namespace GooglePlayInstant.Samples.TestApp
+{
+    /// <summary>
+    /// Tests instant app plugin features through button clicks
+    /// </summary>
+    public class TestApp : MonoBehaviour
+    {
+        private const string CookieEmoticon = "\uD83C\uDF6A";
+        private string _storedCookie;
+
+        /// <summary>
+        /// Sets the instant app cookie to a unique string
+        /// </summary>
+        public void BtnEvt_WriteCookie()
+        {
+            //Write a random value so WriteCookie will always change the cookie
+            var guid = Random.Range(int.MinValue, int.MaxValue);
+            _storedCookie = string.Format("{0}:{1}", guid, CookieEmoticon);
+            CookieApi.SetInstantAppCookie(_storedCookie);

```

Comment:
Check return value and throw exception or print error if false.

Reply:
Done